webfriends-selenium-elements/src/main/java/com/github/bordertech/webfriends/selenium/element/grouping/SStrong.java

```diff
@@ -0,0 +1,19 @@
+package com.github.bordertech.webfriends.selenium.element.grouping;
+
+import com.github.bordertech.webfriends.api.element.grouping.HStrong;
+import com.github.bordertech.webfriends.selenium.common.tag.SeleniumTags;
+import com.github.bordertech.webfriends.selenium.common.tag.TagTypeSelenium;
+import com.github.bordertech.webfriends.selenium.element.AbstractSElement;
+import com.github.bordertech.webfriends.selenium.element.SElement;
+
+/**
+ * Selenium strong element.
+ */
+public class SStrong extends AbstractSElement implements HStrong, SElement {
+
+	@Override
+	public TagTypeSelenium<? extends SStrong> getTagType() {
+		return SeleniumTags.STRONG;
+	}
```

Comment:
This looks like a hang over from SParagraph as well. Not sure why it was implemented like this.

This can be:

``` java
public STagStrong getTagType() {
  return SeleniumTags.STRONG;
}
```

The same change can be made in SParagraph:

``` java
@Override
public STagParagraph getTagType() {
  return SeleniumTags.P;
}
```